djautotask/tests/test_commands.py

```diff
@@ -0,0 +1,73 @@
+import io
+from atws.wrapper import Wrapper
+from django.core.management import call_command
+from django.test import TestCase
+from djautotask.tests import fixtures, mocks, fixture_utils
+
+
+def sync_summary(class_name, created_count):
+    return '{} Sync Summary - Created: {}, Updated: 0'.format(
+        class_name, created_count
+    )
+
+
+def full_sync_summary(class_name, deleted_count):
+    return '{} Sync Summary - Created: 0, Updated: 0, Deleted: {}'.format(
+        class_name, deleted_count
+    )
+
+
+class AbstractBaseSyncTest(object):
+
+    def setUp(self):
+        mocks.init_api_connection(Wrapper)
+
+    def _title_for_at_object(self, at_object):
+        return at_object.replace('_', ' ')
+
+    def _test_sync(self, mock_call, at_object, fixture_list,
+                   full_option=False):
+
+        return_value = fixture_utils.generate_objects(at_object, fixture_list)
+        mock_call(return_value)
+
+        out = io.StringIO()
+        args = ['atsync', at_object.lower()]
+        if full_option:
+            args.append('--full')
+        call_command(*args, stdout=out)
+        return out
+
+    def test_sync(self):
+
+        out = self._test_sync(*self.args)
+        obj_title = self._title_for_at_object(self.args[1])
+
+        self.assertIn(obj_title, out.getvalue().strip())
+
+    def test_full_sync(self):
+        self.test_sync()
```

Comment:
This looks similar to something you fixed in django-connectwise recently where we were running tests multiple times when we didn't intend to, causing errors. Could this cause the same problems or is this different. I see we didn't fix this in the abstractbasesynctest in djc eitehr though.

Reply:
Yeah this is a little similar but the other case was mostly just a lot of poor naming which I thought was resulting in tests running when they shouldn't. I've improved the naming a bit but I think this should be ok for now. 

Might need to change it a bit more when we start adding more classes to sync.